contracts/TracerPerpetualsFactory.sol

```diff
@@ -2,9 +2,9 @@
 pragma solidity ^0.8.0;
 
 import "./Interfaces/ITracerPerpetualSwaps.sol";
-import "./Interfaces/IInsurance.sol";
 import "./Interfaces/ITracerPerpetualsFactory.sol";
 import "./Interfaces/IDeployer.sol";
+import "./Insurance.sol";
```

Comment:
This will work fine for now, but just making a comment that as we flesh out our new contract dependencies, a lot of this deployment stuff will need to change. Eventually the size of the PerpFactory will get too big if we import every fully implemented contract here.